build.gradle.kts

```diff
@@ -1,39 +1,70 @@
 import org.jetbrains.dokka.gradle.DokkaTask
 import org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootExtension
 import org.jetbrains.kotlin.gradle.targets.js.nodejs.NodeJsRootPlugin
+import org.jetbrains.kotlin.gradle.targets.js.npm.tasks.KotlinNpmInstallTask
 
 plugins {
     `maven-publish`
     signing
     id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
-    kotlin("multiplatform") version "1.9.0"
+    kotlin("multiplatform") version "1.9.20"
     id("org.jetbrains.dokka") version "1.8.20"
 }
 
 group = "com.willowtreeapps.opentest4k"
 version = "1.2.4"
 
-val nativeTargets = arrayOf(
-    "linuxX64", "linuxArm64",
-    "macosX64", "macosArm64",
-    "mingwX64",
-    "iosArm32", "iosArm64", "iosX64", "iosSimulatorArm64",
-    "tvosArm64", "tvosX64", "tvosSimulatorArm64",
-    "watchosArm32", "watchosArm64", "watchosX86", "watchosX64", "watchosSimulatorArm64", "watchosDeviceArm64",
-    "androidNativeArm32", "androidNativeArm64", "androidNativeX86", "androidNativeX64",
-)
+val hostOs = System.getProperty("os.name")
 
 kotlin {
+    @Suppress("OPT_IN_USAGE")
+    applyDefaultHierarchyTemplate {
```

Comment:
All targets are kept. Modern (simplier) version of targets hierarchy configuration is used.